test/jasmine/tests/funnel_test.js

```diff
@@ -1326,14 +1326,76 @@ describe('funnel hover', function() {
             .then(done);
         });
 
+        it('should turn off percentages with hoveinfo none or skip', function(done) {
+            gd = createGraphDiv();
+
+            var mock = Lib.extendDeep({}, require('@mocks/text_chart_arrays'));
+            mock.data.forEach(function(t, i) {
+                t.type = 'funnel';
+                t.orientation = 'v';
+                if(i === 0) {
+                    t.hoverinfo = 'none';
+                } else {
+                    t.hoverinfo = 'skip';
+                }
+            });
+
+            function _hover() {
+                var evt = { xpx: 125, ypx: 150 };
+                Fx.hover('graph', evt, 'xy');
+            }
+
+            Plotly.plot(gd, mock)
+            .then(_hover)
+            .then(function() {
+                assertHoverLabelContent({
+                    isEmpty: true
```

Comment:
I'm not a fan of this new `isEmpty: true` option. It makes it look like you're asserting that a given hover label has no content in in - whereas you're asserting that there's no hover label at all.

I would simply use `expect(d3.selectAll('g.hovertext').size()).toBe(0)` here.